.gitignore

```diff
@@ -75,7 +75,6 @@ celerybeat-schedule.*
 *.sage.py
 
 # Environments
-.env
```

Comment:
???

Reply:
.env is already on line 8 in `.gitignore`

Reply:
It's duplicazted (there is also `.env` on line 8)